frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,272 @@
+import React from "react";
```

Comment:
@pavanjoshi914 as per our meeting, why was a new page added? entering your password and then going to a new page to enter it again does not make sense, why wasn't that content on the original page? This also breaks the navigation.

Reply:
we also have this vss and migrate alby hub setting on same page. i didn't felt right to put all this at one place
1. showing mnemonic table
2. vss code logic
3. migrate alby hub logic 

so i created generic backup screen. and separate screen for each process
1. NodeBackup
2. Mmenoic Screen

how this will break navigation? also i decided to remove password in generic backup screen we can just show view recovery phase button there and redirect to the mnemonic screen where we take password input. is that ok?


![image](https://github.com/user-attachments/assets/a634ca07-6afb-4428-8969-08b6321cf039)